keyboards/ergodox_ez/keymaps/rishka/keymap.c

```diff
@@ -0,0 +1,214 @@
+  _______, _______, _______, KC_MS_U, _______, KC_WH_U, _______,
+  _______, _______, KC_MS_L, KC_MS_D, KC_MS_R, KC_WH_D,
+  _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, KC_BTN1, KC_BTN2,
+                                               _______, KC_BTN5,
+                                                        KC_BTN4,
+                                      KC_BTN1, KC_BTN2, KC_BTN3,
+  // right hand
+  _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, KC_WH_U, _______, _______, _______,
+           _______, KC_WH_L, KC_WH_D, KC_WH_R, _______, KC_MPLY,
+  _______, _______, _______, KC_MPRV, KC_MNXT, _______, _______,
+                    KC_VOLU, KC_VOLD, KC_MUTE, _______, _______,
+  _______, _______,
+  _______,
+  KC_ACL2, KC_ACL1, KC_ACL0
+),
+
+[GAME] = LAYOUT_ergodox(
+  // left hand
+  KC_ESCAPE, KC_GRAVE, KC_1, KC_2, KC_3, KC_4, KC_5,
+  KC_F1, KC_Y, KC_Q, KC_W, KC_E, KC_R, KC_T,
+  KC_TAB, KC_LSHIFT, KC_A, KC_S, KC_D, KC_F,
+  KC_LCTRL, KC_B, KC_Z, KC_X, KC_C, KC_V, KC_G,
+  LT(BASE, KC_L), KC_J, KC_I, KC_O, KC_M,
+                                               KC_ESCAPE, _______,
+                                                        KC_ENTER,
+                                      KC_SPACE, KC_LCTRL, KC_LALT,
+ // right hand
+  KC_RGHT,      KC_6,    KC_7,    KC_8,    KC_9,              KC_0,           KC_MINS,
+  TG(SYMB),     KC_Y,    KC_U,    KC_I,    KC_O,              KC_P,           KC_BSLS,
+  KC_H,         KC_J,    KC_K,    KC_L,    LT(MDIA, KC_SCLN), GUI_T(KC_QUOT),
+  MEH_T(KC_NO), KC_N,    KC_M,    KC_COMM, KC_DOT,            CTL_T(KC_SLSH), KC_RSFT,
+  KC_UP,        KC_DOWN, KC_LBRC, KC_RBRC, TT(SYMB),
+  KC_LALT, CTL_T(KC_ESC),
+  KC_PGUP,
+  KC_PGDN, KC_TAB, KC_ENT
+),
+
+[FIRMWARE] = LAYOUT_ergodox(
+  // left hand
+  _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______,
+                                               _______, _______,
+                                                        _______,
+                                      RESET, _______, _______,
+  // right hand
+  _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______,
+           _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______,
+                    _______, _______, _______, _______, _______,
+  _______, _______,
+  _______,
+  _______, _______, EEP_RST
+),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  if (record->event.pressed) {
+    switch (keycode) {
+      case EPRM:
+        eeconfig_init();
+        return false;
```

Comment:
Remove these lines, you're already using `EEP_RST` which is the correct way to do this.

Reply:
updated